src/v0/destinations/splitio/data/EventConfig.json

```diff
@@ -29,5 +29,17 @@
       "type": "toFloat"
     },
     "required": false
+  },
+  {
+    "destKey": "trafficTypeName",
+    "sourceKeys": [
+      "traits.trafficTypeName",
+      "context.traits.trafficTypeName",
+      "properties.trafficTypeName"
+    ],
```

Comment:
Why supporting all, can't we just support if from properties?

Reply:
is `trafficTypeName` user trait or event property?

Reply:
Traffic type can be sent in all of the calls like track, group . Some of them will not have properties. Traffic type is not a user property for sure.